gapic-generator-java/src/main/java/com/google/api/generator/Main.java

```diff
@@ -21,6 +21,7 @@
 import java.io.IOException;
 
 public class Main {
+
```

Comment:
Can we revert this empty line? Same thing for the empty line in `Generator` below.